<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="paletteBut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>show Palette</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="color16">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="rgbText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="cmykText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="xyzText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="labText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rgbLbl">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>111</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rgb</string>
    </property>
   </widget>
   <widget class="QLabel" name="cmykLbl">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cmyk</string>
    </property>
   </widget>
   <widget class="QLabel" name="xyzLbl">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>171</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Xyz</string>
    </property>
   </widget>
   <widget class="QLabel" name="labLbl">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rgbBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmykBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyzBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="labBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn16">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHLS">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HLS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hlsBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="hslText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="hsvText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labHSV">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hsvBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
